client/util/times.utils.js

```diff
@@ -0,0 +1,50 @@
+import moment from 'moment';
+
+export const getTimesBetween = (start, end) => {
```

Comment:
Again, I don't recall using this function in more than one file. It seems kinda pointless to have it in a utilities file for the whole app.